pkg/asset/installconfig/alibabacloud/alibabacloud.go

```diff
@@ -18,11 +18,42 @@ const (
 	defaultAcceptLanguage = "en-US"
 )
 
-var unsupportedRegions = sets.NewString(
-	// Nanjing is a local cloud
-	"cn-nanjing",
+var supportedRegions = sets.NewString(
+	"cn-qingdao",
+	"cn-beijing",
+	"cn-zhangjiakou",
+	"cn-huhehaote",
+	"cn-wulanchabu",
+	"cn-hangzhou",
+	"cn-shanghai",
+	"cn-shenzhen",
+	"cn-heyuan",
+	"cn-guangzhou",
+	"cn-chengdu",
+	"cn-hongkong",
+	"ap-northeast-1",
+	"ap-southeast-1",
+	"ap-southeast-2",
+	"ap-southeast-3",
+	"ap-southeast-6",
+	"ap-southeast-5",
+	"ap-south-1",
+	"us-east-1",
+	"us-west-1",
+	"eu-west-1",
+	"eu-central-1",
+
+	// These regions are local cloud
```

Comment:
Could add a comment where these valid regions were found like https://www.alibabacloud.com/help/en/elastic-container-instance/latest/regions-and-zones. Could then get rid of two comments below.

Reply:
In fact, there is no documentation about it yet, of course I can remove these comments (I prefer it to stay until there is relevant documentation)